src/plugins/intel_gna/src/runtime/floatmath.cpp

```diff
@@ -9,10 +9,6 @@
 #include <cstdint>
 #include <cstdio>
 
-#ifdef __cplusplus
```

Comment:
are you sure about this removal ?

Reply:
For me, it looks like this file was copy-pasted from some library in the early stages of this project and nobody bothered to modify it afterwards. I don't think these functions are actually used anywhere else.

Reply:
this is ifdef and extern C are used to provide C like api( to prevent name mangling).
If you are sure that it is not needed we can remove it.